### ISSUE TITLE
The "Hide right side bar" button disappears when using the zoom of the browser

### ISSUE THREAD

**Issue by valentin-pinkau** [scalableminds/webknossos#5564]
## Context
This bug is present in view mode and tracing mode.

## Steps to Reproduce the bug
1. Open any dataset
2. Use the zoom of the browser (Firefox or Chrome)
3. The "Hide right side bar"-button in the upper right corner disappears

## Your Environment for bug
(Linux Chrome and Firefox) and (Mac Firefox)

**Comment by fm3** [comment]
I didn’t encounter this anymore. @valentin-pinkau Are you still seeing this?

**Comment by valentin-pinkau** [comment]
Yes, I do. But the button is present in some zoom steps and missing in others.
